compose-table/src/main/java/org/dhis2/composetable/ui/DataSetTableUi.kt

```diff
@@ -331,6 +339,16 @@ fun TableCell(
             .border(1.dp, tableCellUiOptions.borderColor(focused))
             .background(tableCellUiOptions.backgroundColor())
     ) {
+        val isNotEditable = cellValue.editable != null && !cellValue.editable
```

Comment:
could we save this if we make `editable` property of `TableCell` **not null**?

Reply:
Not sure why was null, @mmmateos ?

Reply:
Yes, no specific reason, it can be not null

Reply:
This can be null from java in FieldViewModel, should we ignore it?